cmd/runtimecfg/node-ip.go

```diff
@@ -142,7 +166,7 @@ func checkAddressUsable(chosen []net.IP) (err error) {
 	// If using IPv6, verify that the choosen address isn't tentative
 	// i.e. we can actually bind to it
 	if len(chosen) > 0 && net.IPv6len == len(chosen[0]) {
-		_, err = net.Listen("tcp", "["+chosen[0].String()+"]:")
+		_, err = net.Listen("tcp", "["+chosen[0].String()+"]:10010")
```

Comment:
Added it as test and forgot to remove

Reply:
Okay, I think we should remove it so we aren't adding another required port. Since ci has passed on this version, do you want to do that as a followup?